satpy/readers/viirs_l2.py

```diff
@@ -0,0 +1,155 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+# Copyright (c) 2011-2019 Satpy developers
```

Comment:
The years seem wrong, should probably be 2022 or 2022-2023.

Reply:
Updated in c6c97a440